src/rosdep2/main.py

```diff
@@ -461,6 +469,20 @@ def _package_args_handler(command, parser, options, args):
                 print("Skipping non-existent path " + path)
         set_workspace_packages(ws_pkgs)
 
+    # Print installed conflicting or replaced packages
+    if command in ['install', 'check'] and (options.ignore_src or options.from_paths):
+        workspace_pkgs = get_workspace_packages()
+        conflicted_pkgs = get_conflicted_packages()
+        replaced_pkgs = get_replaced_packages()
+
+        for package in workspace_pkgs:
+            if package in conflicted_pkgs:
+                print("WARNING: conflicting installed packages: '{0}' is conflicting '{1}'!".format(
+                    conflicted_pkgs[package], package))
+            if package in replaced_pkgs:
+                print("WARNING: '{0}' replaces '{1}', but '{1}' is already installed!".format(
+                    replaced_pkgs[package], package))
+
```

Comment:
Also, some packages or system dependencies that are marked as "conflicting" or "replaced" might get installed by the package manager (e.g. as a dependency of another non-conflicting package), so you would really have to check the actual "installed" status for all rosdep keys marked as conflicting or replaced after rosdep is finished.